app/views/layouts/_head.html.slim

```diff
@@ -4,3 +4,7 @@ head
 
   = csrf_meta_tags
   = stylesheet_link_tag 'application', media: 'all'
+  link href="images/apple-icon.png" rel="apple-touch-icon" /
+  link href="images/android-icon.png" rel="icon" type="image/png" /
+  link href="images/favicon.png" rel="icon" type="image/png" /
+  link href="/favicon.ico" rel=("shortcut icon") type="image/x-icon" /
```

Comment:
Use = favicon_link_tag instead
You can read more here:
http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/favicon_link_tag